### ISSUE TITLE
Port to SWI Prolog 6

### ISSUE THREAD

**Issue by flowma** [prechac/prechacthis#6]
Apparently, PrechacThis does not run on the current version 6 of SWI-Prolog. That means no installation the desktop, no cross compiling.

**Comment by havarpan** [comment]
just got this codebase to work [here](http://209.38.178.145/test) with swi-prolog 9.2.1 on ubuntu... or maybe it works... at least it seems to work on first sight

**Comment by jjochen** [comment]
Nice 🎉
Found one issue when pressing the back to results link on the details page.

**Comment by havarpan** [comment]
yeah i found it too... just commented out the link haha... but i'll cleanup a bit and publish the updated codebase next week... only one patch in there... the dcg_whitespace didn't work properly

**Comment by havarpan** [comment]
[done](https://github.com/havarpan/neoprechac)
